src/open_inwoner/templates/pages/cases/list_inner.html

```diff
@@ -16,38 +16,41 @@ <h1 class="utrecht-heading-1" id="cases">{{ page_title }} ({{ paginator.count }}
             <div class="card__body">
                 <!-- submission cases -->
                 {% if case.case_type == "OpenSubmission" %}
-                    <a href="{{ case.vervolg_link }}"  class="cases__link">
-                    <p class="h4"><span class="link link__text">{{ case.naam }}</span></p>
+                    <a href="{{ case.vervolg_link }}" class="cases__link" aria-label="{% trans "Bekijk aanvraag" %}" title="{% trans "Bekijk aanvraag" %}">
```

Comment:
Since you changed the `span` element below to "Voltooi aanvraag" (which is a nice way to capture the distinction between cases and open submissions), you should change this too. Also, see previous comment about `aria-label` and `aria-labelledby`.